mods/default/functions.lua

```diff
@@ -717,8 +717,8 @@ end
 
 
 --
--- NOTICE: This method is not an official part of the API yet.
--- This method may change in future.
+-- NOTICE: This methods are not an official part of the API yet.
```

Comment:
These*

Reply:
I think that log_actions() is not an official API function either. On the other hand, i would like to see that. :smiley:

Reply:
aha, sorry. I fixed the typo